### PR TITLE
Create lambda for send emails

### PR DESCRIPTION
**Pull request by 1701722767**

### CODE REVIEW

File: services/notify/functions/email/main.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+class Request:
+    """
+    Request is the class used for handling the request
+    """
+    def __init__(self,message):
+        self.message = message
+        self.err = None
+
+    def validate(self):
```
Comment (nicolascr181) [inline services/notify/functions/email/main.py:23]: If message is a dictionary, this validation code could be improved

File: services/notify/functions/email/main.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+def lambda_handler(event, context):
+    try:
+        _ = context
+        message = getMessage(event)
+        if message == None:
+            return None
+
+        req = Request(message)
+
+        return req.process()
```
Comment (nicolascr181) [inline services/notify/functions/email/main.py:101]: don't forget to return json format
{
 error: "" ,  message: "", data: ""
}

File: services/notify/functions/email/main.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    except Exception as err:
+        print(err)
+
+        return err
```
Comment (nicolascr181) [inline services/notify/functions/email/main.py:106]: same here

File: services/requirements.txt [hunk 4]
```diff
@@ -1,2 +1,3 @@
 PyYAML
 pandas
+boto3
```
Comment (nicolascr181) [inline services/requirements.txt:3]: Is boto3 layer  necessary?,
it's already installed in aws

**Review by nicolascr181 — CHANGES_REQUESTED** [review]

**Review by sergiomonz — APPROVED** [review]

**Review by nicolascr181 — APPROVED** [review]

**Pull request merged**